.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  // "editor.formatOnSave": true,
```

Comment:
remove double quotes from here

Reply:
Its just we can use it if we want

Reply:
ok, you have to change it for now to merge it